src/components/Dashboard/CustodianListSm.vue

```diff
@@ -125,7 +118,7 @@ $avatar-width: 2.75rem;
   padding: 0;
 }
 
-.v-list-item:first-of-type {
+.v-list-item:not(last-of-type) {
```

Comment:
Changed from adding bottom padding on first element to adding bottom padding on all-but-the-last element (for situations where there are more than 2 elements).